conda/recipes/libraft/meta.yaml

```
@@ -66,21 +81,34 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
         - librmm
     requirements:
       run:
         - {{ pin_subpackage('libraft-headers-only', exact=True) }}
-        - cuda-profiler-api {{ cuda_profiler_api_run_version }}
         - librmm ={{ minor_version }}
-        - libcublas {{ libcublas_run_version }}
-        - libcublas-dev {{ libcublas_run_version }}
-        - libcurand {{ libcurand_run_version }}
-        - libcurand-dev {{ libcurand_run_version }}
-        - libcusolver {{ libcusolver_run_version }}
-        - libcusolver-dev {{ libcusolver_run_version }}
-        - libcusparse {{ libcusparse_run_version }}
-        - libcusparse-dev {{ libcusparse_run_version }}
+        - cuda-version {{ cuda_spec }}
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        - cuda-profiler-api {{ cuda11_cuda_profiler_api_run_version }}
+        - libcublas {{ cuda11_libcublas_run_version }}
+        - libcublas-dev {{ cuda11_libcublas_run_version }}
+        - libcurand {{ cuda11_libcurand_run_version }}
+        - libcurand-dev {{ cuda11_libcurand_run_version }}
+        - libcusolver {{ cuda11_libcusolver_run_version }}
+        - libcusolver-dev {{ cuda11_libcusolver_run_version }}
+        - libcusparse {{ cuda11_libcusparse_run_version }}
+        - libcusparse-dev {{ cuda11_libcusparse_run_version }}
+        {% else %}
+        - cuda-cudart
```

This doesn't match the math libraries, which are all `-dev` packages. I think we might want:
```suggestion
        - cuda-cudart-dev
```